sag_ji1.par

```diff
@@ -1,34 +1,34 @@
-! Athenian Sagittal JI notation. Dave Keenan & George Secor. 2007-Nov-29
+! Athenian Sagittal JI notation. Dave Keenan & George Secor. 2007-Nov-29, updated 2020-Apr-08
 ! The average step is ln(3^7/2^11)/ln(2)*1200/21 = 5.413571717 cents
 !
 max_abs_slope     8.28
 c_exp_range       -6   10
 nc_exp_range      -1    5
 !
 !Lower bound    Symbol  Value     Short  21-EDA
--68.06462809      (!/   8192/8505   w    -12
--62.46546581      (!)   704/729     o    -11
--56.84250303      \!/   32/33       v    -10
--51.21954025      \!)   35/36       u     -9
--45.62037797     \\!    6400/6561   _     -8
--40.25327468      (!(   44/45       d     -7
--35.11809146      (!    45056/45927 j     -6
--30.00698658       !)	63/64       t     -5
--24.63988329      \!    80/81       \     -4
--18.78486152      ~!(   4096/4131   a     -3
--13.41775823      )!(   891/896     i     -2
--8.050654935       !(   5103/5120   c     -1
--2.683551645
-2.683551645        |(   5120/5103   r      1
-8.050654935       )|(   896/891     *      2
-13.41775823       ~|(   4131/4096   g      3
-18.78486152       /|    81/80       /      4
-24.63988329        |)   64/63       f      5
-30.00698658       (|    45927/45056 ?      6
-35.11809146       (|(   45/44       q      7
-40.25327468      //|    6561/6400   =      8
-45.62037797       /|)   36/35       n      9
-51.21954025       /|\   33/32       ^     10
-56.84250303       (|)   729/704     @     11
-62.46546581       (|\   8505/8192   m     12
-68.06462809
+-68.084530821955000      (!/   8192/8505   d    -12

```

Comment:
The lower bounds values increased accuracy to 15 digits, in all of the JI files. This was what George's JI calculator supported (not all of them seem to go to that level of accuracy, but at least one does). The increased accuracy doesn't not appear to break Scala. 

Every one of them changed slightly too. This appears to correspond to the change between the first and second version of the JI Notation Spreadsheet.

Reply:
Good. Probsbly should add "Medium precision" in heading somewhere.

Reply:
I have not yet propagated those 4 changes to these config files, no. I don't think we've 100% settled on the 28th mina change. Still going back and forth on whether to snap to a comma mean or an EDA midpoint.

Reply:
That's fine. I think it's a good idea to leave all those changes out for now.